pyartcd/pyartcd/pipelines/release_readiness/checks/build_sync.py

```
@@ -0,0 +1,124 @@
+        ocp_version (str): OCP version string (e.g. "4.21").
+
+    Return Value(s):
+        CheckResult: Build-sync check result.
+    """
+
+    _LOGGER.info("Checking build-sync status for %s", ocp_version)
+
+    try:
+        try:
+            await asyncio.to_thread(jenkins.init_jenkins)
+        except Exception:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)